css-align-3/content-distribution/place-content-shorthand-002.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<title>CSS Box Alignment: place-content shorthand - multiple values specified</title>
+<link rel="author" title="Javier Fernandez" href="mailto:jfernandez@igalia.com" />
+<link rel="help" href="http://www.w3.org/TR/css3-align/#propdef-place-content" />
+<meta name="assert" content="Check that setting two values to place-content sets the first one to 'align-content' and the second one to 'justify-content'." />
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="../resources/alignment-parsing-utils.js"></script>
+<div id="log"></div>
+<script>
+    var values1 = ["normal"].concat(position.slice(1,5), distribution.slice(1,3), baseline.slice(1,2));
+    var values2 = position.slice(5).concat(distribution.slice(3), baseline.slice(2));
```

Comment:
I don't see a big benefit in not testing all the combinations, now that we've everything in place.
Actually these lines of code were confusing for me as I was not sure why you were not testing the whole thing.
So, IMHO I'd go and test all the possible combinations.